src/store/actions.ts

```diff
@@ -1080,7 +1080,7 @@ export const submit: any = async ({ commit, getters }): Promise<any> => {
       }
       request = await putNameReservation(getters.getNrId)
     }
-    if (request) await paymentModule.togglePaymentModal(true)
+    if (request) await dispatch('togglePaymentModal', true)
```

Comment:
OK, tell me the different between dispatch and a commit or action?

Reply:
So, from what i've read..
-Dispatch: To call an action ( after the next tick ) - Asynchronous
- Commit: When you are ready to commit a value to store. - Synchronous 

That being said. We have these compound methods that are doing both... 
I'm not entirely sure they are 'ACTIONS' but it is probably partially true.

Reply:
Why did you choose dispatch here?

Reply:
Because `togglePaymentModal` is an action within the payment module.

Reply:
Oh, so that's how you call an action in another module? Gotcha.